static/js/tools/statvar_menu/util.tsx

```diff
@@ -48,7 +48,7 @@ function getStatsVarInfo(
 async function getStatsVar(
   dcids: string[],
   sample = false,
-  sampleSize = 50
+  sampleSize = 5
```

Comment:
add some comments about the background of this constant?

Reply:
done